benchmarks/dashboard-stack/lib/dashboard-stack.ts

```diff
@@ -1,22 +1,67 @@
 import * as cdk from '@aws-cdk/core';
 import * as cloudwatch from '@aws-cdk/aws-cloudwatch';
 import * as iam from '@aws-cdk/aws-iam';
+import * as _lambda from '@aws-cdk/aws-lambda';
+import * as codebuild from '@aws-cdk/aws-codebuild';
+import * as events from '@aws-cdk/aws-events';
+import * as targets from '@aws-cdk/aws-events-targets';
+import * as s3 from '@aws-cdk/aws-s3';
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as s3deploy from '@aws-cdk/aws-s3-deployment'
 import * as path from 'path';
 import * as fs from 'fs';
 
 export class DashboardStack extends cdk.Stack {
     constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
         super(scope, id, props);
 
-        const benchmark_config_json = fs.readFileSync(path.join(__dirname, 'benchmark-config.json'), 'utf8');
+        const benchmark_config_json_path = path.join(
+            __dirname,
+            "..",
+            "..",
+            "benchmark-config.json"
+        );
+        const benchmark_config_json = fs.readFileSync(benchmark_config_json_path, 'utf8');
         const benchmark_config = JSON.parse(benchmark_config_json);
 
+        // Lambda to handle trigger codebuild to deploy benchmark and clean up benchmark after tests
+        const lambda_role = iam.Role.fromRoleArn(this, 'LambdaRole', 'arn:aws:iam::123124136734:role/lambda-role-CFN');
```

Comment:
Role is hardcoded to our account--any way we can un-harcode it? :)

Reply:
Following PR.